packages/cli/src/commands/add/index.ts

```diff
@@ -19,20 +19,29 @@ export default async function add(
   cwd: string,
   { empty, open }: { empty?: boolean; open?: boolean },
   config: Config
-) {
+): Promise<void> {
   const packages = await getPackages(cwd);
   if (packages.packages.length === 0) {
     throw new Error(
       `No packages found. You might have ${packages.tool} workspaces configured but no packages yet?`
     );
   }
+
   const versionablePackages = packages.packages.filter(
     (pkg) =>
       !shouldSkipPackage(pkg, {
         ignore: config.ignore,
         allowPrivatePackages: config.privatePackages.version,
       })
   );
+
+  if (versionablePackages.length === 0) {
+    info("No versionable package detected");
+    info('- Check "ignore" setting info configuration');
+    info('- Ensure that relevant package.json file(s) have "version" field');
+    return;
+  }
```

Comment:
I wonder if we should be throwing an error here like above, since it seems unexpected that there's no versionable packages to add changesets for.

Maybe something like this? (update the wording too, let me know what you think)
```suggestion
  if (versionablePackages.length === 0) {
    error("No versionable packages found");
    error('- Ensure the packages to version are not in the "ignore" option');
    error('- Ensure that relevant package.json files have the "version" field');
    throw new ExitError(1);
  }
```

Reply:
I thought about throwing an error, but IMHO it was too "scary" (Don't know a better term 😢).
But if you handle similar scenarios using errors where you throw the error I will update the code and the test accordingly.

Reply:
Additional thought: if changesets are created via automated tool, have no versionable packages should trigger an error to interrupt any following command / tasks.

In the end, I agree with you, an error is better to handle this scenario, I'll the code to issue an error.